Core/Core/Api/GraphQL/Client.GraphqlClientOperations.cs

```diff
@@ -543,16 +543,16 @@ public Task<bool> StreamGrantPermission(StreamGrantPermissionInput permissionInp
     /// <param name="cancellationToken"></param>
     /// <param name="permissionInput"></param>
     /// <returns></returns>
-    public async Task<bool> StreamGrantPermission(CancellationToken cancellationToken, StreamGrantPermissionInput permissionInput)
```

Comment:
it's prob a good idea to keep both `StreamGrantPermission` and `StreamUpdatePermission` with a server version check in each as this is p breaking and people are unlikely to upgrade their servers right away